ci-operator/step-registry/assisted/common/setup/assisted-common-setup-commands.sh

```diff
@@ -0,0 +1,268 @@
+#!/bin/bash
```

Comment:
i am not sure i understand the logic, we create ansible playbook and then run it? what's the point? In case we moved to ansible we should use some templates or already written playbook and not creating it on the fly in bash script as it makes it not readable at all

Reply:
Ansible is more structure and force you to well document your code (It has a name for every task). We already have Ansible for the packet machine creation so it's not new technology.